crates/runtime/src/datafusion.rs

```diff
@@ -429,7 +433,9 @@ impl DataFusion {
                 self.register_federated_table(&dataset, data_connector, federated_read_table)
                     .await?;
             }
-            Table::View(sql) => self.register_view(dataset_table_ref.clone(), sql)?,
+            Table::View(_) => unreachable!(
```

Comment:
Do we even need this enum variant anymore? The whole point of it was to register the view here.

Reply:
Updated, thank you